docs/overview.rst

```diff
@@ -29,50 +32,55 @@ The crash ingestion pipeline that we have at Mozilla looks like this:
       subgraph stores {
          rank=same;
 
-         aws [shape=tab, label="S3", style=filled, fillcolor=gray];
+         s3raw [shape=tab, label="S3 (Raw)", style=filled, fillcolor=gray];
+         pigeon [shape=cds, label="pigeon"];
+         rabbitmq [shape=tab, label="RMQ", style=filled, fillcolor=gray];
       }
 
       subgraph proc {
```

Comment:
It wasn't changed in this PR, but making the arrows to Postgres bidirectional to represent data being written to them would be helpful, as it would be more consistent with arrows representing data flowing from one place to another.